client/src/common/styledComponents/List.js

```diff
@@ -4,10 +4,14 @@ import styled from 'vue-styled-components';
 export const List = styled.div`
   display: flex;
   flex-wrap: wrap;
+
+  & > * {
```

Comment:
Some magic to our app)

Reply:
Just made it possible to apply flex if list item is wrapped into something, (router link for example), it might be a wrong place to do so I'll check